samples/java/src/main/java/com/example/Sample.java

```diff
@@ -51,13 +57,37 @@ public static void main( String[] args ) throws Exception
 
             String time = LocalDateTime.now().toString();
 
+            // Submit a transaction, blocking until the transaction has been committed on the ledger.
             System.out.println("Submitting transaction to basic chaincode with value " + time + "...");
             byte[] result = contract.submitTransaction("put", "time", time);
             System.out.println("Submit result = " + new String(result));
 
             System.out.println("Evaluating query...");
             result = contract.evaluateTransaction("get", "time");
             System.out.println("Query result = " + new String(result));
+
+            // Submit multiple transactions asynchronously
+            List<Supplier<TransactionPackage.TxValidationCode>> commits = new ArrayList<>();
+            for (int i = 0; i < 5; i++) {
+                Transaction txn = contract.newProposal("put")
+                        .addArguments("key" + i, "value" + i)
+                        .build()
+                        .endorse();
+                System.out.println("Proposal result = " + new String(txn.getResult()));
+                Supplier<TransactionPackage.TxValidationCode> commit = txn.submitAsync();
```

Comment:
The `Supplier<TransactionPackage.TxValidationCode>` is equivalent to the `Commit` in Go and Node. It should probably be a `Commit` and not a plain `Supplier` but regardless... this code should be:
```
Transaction txn = contract.newProposal("put")
                        .addArguments("key" + i, "value" + i)
                        .build()
                        .endorse();
                Supplier<TransactionPackage.TxValidationCode> commit = txn.submitAsync();
                System.out.println("Proposal result = " + new String(txn.getResult()));
```
So the result is printed after submit to the orderer is successful to be consistent with the Go and Node samples.